packages/terra-search-field/src/SearchField.jsx

```diff
@@ -268,7 +268,6 @@ class SearchField extends React.Component {
     );
 
     const inputAriaLabelText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
-    const groupNameValue = groupName === 'Search' ? intl.formatMessage({ id: 'Terra.searchField.search' }) : groupName;
```

Comment:
This line allows users to set custom aria-label via groupName prop. Removing this breaks that functionality. Pls see this example on SR https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/search-field/examples/search-field-with-no-label. Before and after your change. It should announce the custom aria label

Reply:
Passing "groupName" prop to input field that should announce the custom visual label.